tools/tidb-binlog-cluster.md

```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
         pump3 ansible_host=172.16.10.74 deploy_dir=/data1/pump
         ```
 
-2. 部署并启动 TiDB 集群
-
-    使用 Ansible 部署 TiDB 集群的具体方法参考 [TiDB Ansible 部署方案](../op-guide/ansible-deployment.md)，开启 binlog 后默认会部署和启动 Pump 服务。
+2. 部署并启动含 pump 组件的 TiDB 集群
+    
+    参照上文配置完 inventory.ini 后，选择一种方式进行部署
+    
+    （a）在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行
+         
+         // 1）部署 pump_servers 和 node_exporters
```

Comment:
- The "//" characters in this part make it like a code block, which is weird. Please remove them.
- Use inline code format or code block format only for code.